Components/Card/card.css

```diff
@@ -0,0 +1,202 @@
+@import url("../../main.css");
+@import url('../../colors.css');
+@import url('../../typography.css');
+
+.container {
+    max-width: var(--width-xxxs);
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin: var(--space-sm);
+    box-shadow: RGBA(0, 0, 0, 0.24) 0px 3px 8px;
+    border-radius: var(--space-xxs);
+}
+
+.content {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    position: relative;
+}
+
+.content-row {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    position: relative;
+}
+
+/* titles */
+
+.title {
+    padding: var(--space-xs);
+}
+
+.title-top {
+    background-color: var(--primary-color);
+    color: white;
+    padding: var(--space-xs);
+}
+
+.title-overlay-bottom {
+    position: absolute;
+    bottom: 0%;
+    width: 100%;
+    color: white;
+    padding: var(--space-xs);
+}
+
+.title-overlay-bottom>i {
+    position: absolute;
+    z-index: 2;
+    left: 90%;
+    bottom: 0%;
+    font-size: var(--header2);
+    color: white;
+}
+
+.title-overlay-mid {
+    position: absolute;
+    bottom: 40%;
+    text-align: center;
+    width: 100%;
+    z-index: 2;
+    color: white;
+}
+
+/* utilites */
+
+.product-description {
+    color: var(--error-color);
+}
+
+.product-price {
+    margin-top: 10px;
+}
+
+.full-width {
+    width: 100%;
+}
+
+/* margin for content of card */
+
+.about {
+    margin: var(--space-xs);
+}
+
+/* image to occupy mxaim size of the container */
+
+img {
+    width: 100%;
+}
+
+/* fixed sixe of small image for smaller cards*/
+
+.img-size {
+    width: 125px;
+}
+
+/* bages */
+
+.badge>i {
+    position: absolute;
+    z-index: 2;
+    left: 82%;
+    font-size: var(--header2)
+}
+
+.badge>h4 {
+    position: absolute;
+    z-index: 2;
+    left: 0%;
+    top: 20%;
+    font-size: var(--header2);
+    padding: 10px;
+    background-color: var(--primary-color);
+    color: white;
+}
+
+/* blur effect on images of cards */
+
+.blur {
+    z-index: 1;
+    opacity: 1;
+    filter: blur(3px);
+}
+
+.blur:hover {
+    opacity: 1;
+    z-index: 1;
+    filter: blur(0px);
+}
+
+/* in-stock colors */
+
+#colors {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
```

Comment:
This is also a default value. Attaching link for reference 

https://css-tricks.com/almanac/properties/j/justify-content/#:~:text=flex%2Dstart%20(default)%3A%20items%20are%20packed%20toward%20the%20start%20line

Reply:
👍🏼